.tina/collections/global.tsx

```diff
@@ -239,5 +239,71 @@ export const globalSchema: Collection = {
         },
       ],
     },
+    {
+      type: "object",
+      name: "jotForm",
+      label: "JotForm",
```

Comment:
is this a component I can drop on any page?

Reply:
Fixed - added a component that can be reused